.github/workflows/ci.yaml

```diff
@@ -8,25 +8,23 @@ on:
       - main
 
 jobs:
-  lint:
-    strategy:
-      fail-fast: false
-      matrix:
-        os: [ 'ubuntu-22.04' ]
-
-    name: ${{ matrix.os }} / Lint
```

Comment:
Can we keep the name and update it to Go Lint instead? The default job names are not as neat in the GitHub UI.

Reply:
<img width="770" alt="Screenshot 2023-07-20 at 12 00 47 PM" src="https://github.com/aws/shim-loggers-for-containerd/assets/8747309/e6784867-5485-4879-b9c6-5279a63dd075">

I got your idea of trying to make the naming like CI/<Platform>/... but I'm thinking there can be multiple "platformless" CI jobs including multiple linters. I feel the current pattern is not so inclusive for them.

I could rename the other CI jobs as well to make it consistently later and eventually the naming can be similar as nerdctl.

Reply:
Okay, just would prefer it to be consistent such that at a quick glance I can understand what is failing. Even nerdctl isn't consistent. e.g. https://github.com/containerd/nerdctl/pull/2383